tests/test_task.py

```diff
@@ -394,6 +394,12 @@ class SubclassTask(Task):
     assert t2.argo_task.dependencies[0] == 't'
 
 
+def test_supply_args():
+    t = Task('t', args=["arg"])
+    assert t.argo_template.container.args == ["arg"]
+    assert 'command' not in t.argo_template.container
+
+
```

Comment:
What do you think about an assertion inside an existing test for the presence of `["python]` when no command and no args are specified? 🙂

Reply:
This gets tested in test `test_task_command_parses` on line 203